go/core/tracing/tracing.go

```diff
@@ -185,3 +185,8 @@ func SetCustomMetadataAttr(ctx context.Context, key, value string) {
 func SpanPath(ctx context.Context) string {
 	return spanMetaKey.FromContext(ctx).Path
 }
+
+// SpanMetadata returns the span metadata
+func SpanMetadata(ctx context.Context) *spanMetadata {
```

Comment:
Want to set the input for the new span used in defineEvaluator. Is this okay?

Reply:
Input is the second to last arg in `RunInNewSpan`, I see you have it as something else right now. Does that not work for you?